apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue

```diff
@@ -33,23 +33,26 @@ import {
 } from '@spaceone/design-system';
 import type { MenuItem } from '@spaceone/design-system/types/inputs/context-menu/type';
 
-import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
-
 import { i18n } from '@/translations';
 
-import ErrorHandler from '@/common/composables/error/errorHandler';
-
 import { useCollectorFormStore } from '@/services/asset-inventory/collector/shared/collector-forms/collector-form-store';
 
 const collectorFormStore = useCollectorFormStore();
 const collectorFormState = collectorFormStore.$state;
 
+const props = defineProps<{
+    getVersionsOnPluginIdChange?: boolean;
+}>();
+
 const emit = defineEmits<{(event: 'update:isVersionValid', value: boolean): void;
 }>();
 
 const state = reactive({
     pluginId: computed<string|undefined>(() => collectorFormStore.pluginId),
-    versions: [] as MenuItem[],
+    versionItems: computed<MenuItem[]>(() => collectorFormState.versions.map((value, index) => {
+        if (index === 0) return { type: 'item', label: `${value} (latest)`, name: value };
+        return { type: 'item', label: value, name: value };
```

Comment:
question: Is `versions` always sorted? 🧐

Reply:
yes!